packages/react-vapor/src/components/validation/hoc/tests/WithNonEmptyValueMultiSelectValidationHOC.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import {mountWithStore} from 'enzyme-redux';
+import React from 'react';
+import {act} from 'react-dom/test-utils';
+import {composeMockStore, getStoreMock, withSelectedValues} from '../../../../utils/tests/TestUtils';
+import {IMultiSelectOwnProps, MultiSelectConnected} from '../../../select/MultiSelectConnected';
+import {ValidationActions} from '../../ValidationActions';
+import {ValidationTypes} from '../../ValidationTypes';
+import {
+    IWithNonEmptyValueMultiSelectValidationProps,
+    withNonEmptyMultiSelectHOC,
+} from '../WithNonEmptyValueMultiSelectValidationHOC';
+
+const MultiSelectWithNonEmpty = withNonEmptyMultiSelectHOC(MultiSelectConnected);
+
+describe('MultiSelectWithNonEmpty', () => {
+    const defaultProps: IMultiSelectOwnProps & IWithNonEmptyValueMultiSelectValidationProps = {
+        id: 'SOME_ID',
+        validationMessage: 'ohno',
+        items: [],
+    };
+
+    let store: ReturnType<typeof getStoreMock>;
+
+    afterEach(() => {
+        store.clearActions();
+    });
+
+    const ONE_VALUE = '🐟';
+
+    it('should through an error is there are no selection', () => {
```

Comment:
"should through an error is there" 🤔 two typos here 😄  

Maybe:

```suggestion
    it('should dispatch a setError action on mount if there are no values selected', () => {
```

You need to revise the other test names as well :P